src/os/resourcesdir.zig

```diff
@@ -54,11 +55,16 @@ pub fn resourcesDir(alloc: std.mem.Allocator) !?[]const u8 {
             }
         }
 
-        // On all platforms, we look for a /usr/share style path. This
+        // On all platforms (except BSD), we look for a /usr/share style path. This
         // is valid even on Mac since there is nothing that requires
         // Ghostty to be in an app bundle.
         inline for (sentinels) |sentinel| {
-            if (try maybeDir(&dir_buf, dir, "share", sentinel)) |v| {
+            if (try maybeDir(
+                &dir_buf,
+                dir,
+                if (builtin.target.os.tag == .freebsd) "local/share" else "share",
```

Comment:
This seems wrong. If this path is relative to the installation prefix, then just "share" is correct.

It is the PREFIX value that is different between FreeBSD and Linux (/usr/local vs. /usr), but not the layout inside.